guides/spin/canary-configs/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title:  "Manage Canary Configs"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+`spin` can manage the whole lifecycle of your canary configs:
+
+```bash
+$ spin canary canary-configs
+Usage:
+   canary canary-config [command]
+
+Aliases:
+  canary-config, canary-configs, cc
+
+Available Commands:
+  delete      Delete the provided canary config
+  get         Get the canary config with the provided id
+  list        List the canary configs
+  retro       Retro the provided canary config
+  save        Save the provided canary config
+
+Flags:
+  -h, --help   help for canary-config
+
+Global Flags:
+      --config string            path to config file (default $HOME/.spin/config)
+      --default-headers string   configure default headers for gate client as comma separated list (e.g. key1=value1,key2=value2)
+      --gate-endpoint string     Gate (API server) endpoint (default http://localhost:8084)
+  -k, --insecure                 ignore certificate errors
+      --no-color                 disable color (default true)
+      --output string            configure output formatting
+  -q, --quiet                    squelch non-essential output
+
+Use " canary canary-config [command] --help" for more information about a command.
+```
+
+The following assumes Spinnaker is running and Gate is listening on `http://localhost:8084`. If
+gate is running elsewhere, you can set the Gate endpoint with the global `--gate-endpoint` flag.
+
+## Managing Your Canary Config's Lifecycle
+
+### Create and update canary configs with `save`
+
+```bash
+$ spin canary canary-configs save --file <path to canary config json>
+```
+
+Note that `save` accepts the canary config in JSON format. You can quickly export an existing
```

Comment:
IIUC...

Would it work to say
`$ spin canary canary-configs save|get --file <path to canary config json>`

And then the explanation would be
Where:
* `save` accepts the canary config in JSON format from the file identified using the `--file` argument
* `get` exports an existing canary config into the identified file.

Reply:
Care to make these in a follow-up? We could also make similar changes in `pipeline-templates`, etc.

Reply:
Yeah, I can do that.

Reply:
Just to clarify also:

> get exports an existing canary config into the identified file.

`get` just dumps the json to stdout, but you can save that in a file if you want. There's no explicit output file for `get`.